spec/order_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'order'
+
+describe Order do
+
+	let (:order){double(:order)}
+	it 'should take orders with 2 arguments' do
+		expect(subject).to respond_to(:new_order).with(2).argument
+	end
+
+	it 'should take order and add it to order' do
+		subject.new_order([[2,2],[1,1]],20)
+		expect(subject.order).to eq({:FishandChips=>14, :Burger=>6})
```

Comment:
Symbols should use lowercase and be underscored (`:fish_and_chips` and `:burger` here)